ui/src/components/MainContainer.js

```diff
@@ -244,6 +245,12 @@ const MainContainer = () => {
           permissions={['ministry', 'tc', 't3ch', 'admin', 'stewardship']}
           component={AdminAnnouncementContainer}
         />
+        <PrivateRoute
+          exact
+          path="/admin/faqs"
+          permissions={['t3ch', 'admin', 'stewardship']}
```

Comment:
should we give this access to tc too? in case they are submitting FAQ for their crew? @ksuhartono97 
which makes me think we should probably make a guide for this if that's the case l o l